locales/en.json

```diff
@@ -1,10 +1,10 @@
 {
     "Title": "Contoso Air",
   "Home": {
-    "HeroMessage": "Fly. Feed your heart."
+    "HeroMessage": "Some new Hero Mesage."
```

Comment:
this isn't QUITE right.  Can we make it "Fly, fly, fly!"

Reply:
Made the change.